internal/runners/initialize/init.go

```diff
@@ -161,23 +186,12 @@ func (r *Initialize) Run(params *RunParams) (rerr error) {
 		}
 	}
 
-	// Match the case of the organization.
-	// Otherwise the incorrect case will be written to the project file.
```

Comment:
Can we please keep this comment so that we don't lose the context? Otherwise `getOwner()` looks a bit convoluted with the loops.